_posts/en/newsletters/2022-11-16-newsletter.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Bitcoin Optech Newsletter #226'
+permalink: /en/newsletters/2022/11/16/
+name: 2022-11-16-newsletter
+slug: 2022-11-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to enable generalized smart
+contracts on Bitcoin and summarizes a paper about addressing LN channel
+jamming attacks.  Also included are our regular sections with
+descriptions of changes to services and client software, announcements
+of new releases and release candidates, and summaries of notable changes
+to popular Bitcoin infrastructure software.
+
+## News
+
+- **General smart contracts in Bitcoin via covenants:** Salvatore Ingala
+  [posted][ingala matt] to the Bitcoin-Dev mailing list a proposal for a
+  new type of [covenent][topic covenants] (requiring a soft fork) that
+  would allow using [merkle trees][] to create smart contracts that can
+  carry state from one onchain transaction to another.  To use an
+  example from Ingala's post, two users could wager on a game of chess
+  where the contract could hold the rules of the game and the state
+  could hold the positions of all the pieces on the board, with it being
+  possible to update that state by each onchain transaction.  Of course,
+  a well-designed contract would allow the game to be played offchain
+  with only the settlement transaction at the end of the game being put
+  onchain (or possibly even then staying offchain if the game was played
+  within another offchain construct, such as a payment channel).
+
+    Ingala explains how the work could help design [joinpools][topic
+    joinpools], opportunistic rollups (see [Newsletter #222][news222
+    rollup]), and other stateful constructions.
+
+- **Paper about channel jamming attacks:** Clara Shikhelman and Sergei
+  Tikhomirov [posted][st unjam post] to the Lightning-Dev mailing list
+  the summary of a [paper][st unjam paper] they've written about
+  solutions to [channel jamming attacks][topic channel jamming attacks].
+  These attacks, first described in 2015, can make large numbers of
+  channels unusable for long periods of time at negligible cost to an
+  attacker.
+
+    The authors split jamming attacks into two types. The first is *slow
+    jamming* where a channel's limited slots or funds for payment
+    forwarding are made unavailable for long periods of time---which
+    rarely happens with legitimate payments.  The second type is *fast
+    jamming* where the slots and funds are blocked only briefly---which
+    happens often with normal payments, potentially making fast jamming
+    harder to mitigate.
```

Comment:
```suggestion
    harder both to detect and mitigate.
```
(but check my understanding)

Reply:
I thought about phrasing it like this myself, but none of the solutions for jamming attacks I'm aware of make any attempt at detection.  This is probably important for preserving privacy on onion-wrapped payments.  The solutions focus on trying to increase the cost of unwanted network uses as high as possible while keeping the cost of wanted uses as low as possible.

Leaving this text unchanged.